install.ps1

```diff
@@ -51,7 +51,7 @@ else {
 $PIP_INSTALL = """$NNI_PYTHON3\python"" -m pip install "
 
 if (!(Test-Path $NNI_DEPENDENCY_FOLDER)) {
-    New-Item $NNI_DEPENDENCY_FOLDER -ItemType Directory
+    $null = New-Item $NNI_DEPENDENCY_FOLDER -ItemType Directory
```

Comment:
Why add `$null`?

Reply:
The original command, `New-Item $NNI_DEPENDENCY_FOLDER -ItemType Directory`, will output command result to console. The default output is like below, if there are a lot of this kind of information, it buries important information. Use $null can put stdout to null pipeline. If there is error, it can show on screen as red text also, not be send to null pipeline.

```
    Directory: D:\temp


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----          6/8/2020  11:30 AM                test1
```

Reply:
ok